canonical-web-and-design/cloud-init.io
# Update global nav and add cookie policy

## Done

- Update global nav and add cookie policy

## QA

1. Check out this feature branch
2. Run the site using the command ./run serve
3. View the site locally in your web browser at: http://0.0.0.0:8009
4. Run through the following QA steps
5. See the new cookie popup and try the settings
6. Test the "Manage your tracker settings" link from the footer

## Issue / Card

Fixes #148
